common/components/internal-header/_internal-header.scss

```diff
@@ -1,311 +1,182 @@
-$govuk-header-background: govuk-colour("black");
+$govuk-header-background: govuk-colour('black');
 $govuk-header-border-color: $govuk-brand-colour;
 $govuk-header-border-width: govuk-spacing(2);
-$govuk-header-text: govuk-colour("white");
-$govuk-header-link: govuk-colour("white");
-$govuk-header-link-hover: govuk-colour("white");
+$govuk-header-text: govuk-colour('white');
+$govuk-header-link: govuk-colour('white');
+$govuk-header-link-hover: govuk-colour('white');
 $govuk-header-link-active: #1d8feb;
 $govuk-header-nav-item-border-color: #2e3133;
 
 .app-header {
-  @include govuk-font($size: 16);
-
-  color: $govuk-header-text;
-  background: $govuk-header-background;
-}
-
-.app-header__container--full-width {
```

Comment:
Are we removing support for this modifier with these changes? Might be nice to see if we can still support it.

Reply:
From what I could see, it wasn't used anywhere in the project, the styles that it added are included in `.app-header__container` now (except for border-color, which had no effect outside of `container--with-border`, which also wasn't used anywhere)